app/src/main/java/ch/epfl/skysync/components/FlightCalendar.kt

```diff
@@ -32,15 +37,13 @@ import java.time.LocalDate
  */
 @Composable
 fun FlightTile(date: LocalDate, time: TimeSlot, flight: Flight?, onClick: () -> Unit) {
-  var weight = 0.5f
-  if (time == TimeSlot.PM) {
-    weight = 1f
-  }
+  val testTag = date.toString() + time.toString()
+  val weight = if (time == TimeSlot.AM) 0.5f else 1f
   if (flight != null) {
     Button(
```

Comment:
I wonder if using "Card" rather than "Button" would be more appropriate but anyways it works here

Reply:
Yes, it's probably better, but it's from the previous version of the calendar and as it seemed to work, I didn't want to change it.